src/buildHTML.js

```diff
@@ -50,6 +50,23 @@ const isBinRightCanceller = function(node, isRealGroup) {
 };
 
 /**
+ * Splice out any spaces from `children` starting at position `i`, and return
+ * the spliced-out array. Returns null if `children[i]` does not exist or is not
+ * a space.
+ */
+const spliceSpaces = function(children, i) {
+    let j = i;
```

Comment:
Perhaps it would be more intuitive to give `i` a default value of 0, via `let j = i || 0`?